### DISCUSSION TITLE
Availability of genereated planet Mbtile

### DISCUSSION THREAD

**Discussion by utagawal** [onthegomap/planetiler#286] [Ideas]
Hi,

Thanks a lot for sharing this great work.

I assume you are using it to generate the planet MBTile on a regular basis for your Onthegomap project with planetiler.

I'm looking for a preprocessed, regularly updated map for a side project too (for Mountain Biking :-) ), I was wondering if there would any plans for you to make your planet generated mbtiles files available for others to use (crediting your work to avoid multiple people consuming computing power / electricity / CO2 to generate the same file ) ?

**Comment by ZeLonewolf** [comment]
@utagawal I maintain a planet.mbtiles on an AWS EFS share for the OpenStreetMap Americana project.  If you're on AWS, I'd be happy to grant read access to it.

**Reply by msbarry** [reply]
@wipfli was also experimenting with putting an mbtiles build in a requester-pays s3 bucket but we realized it costs $8 in egress fees to get the file out of aws, but only <$1 in compute to rebuild it from the source. Not sure about resource consumption though. 

There also a few companies that you can pay for a plan that lets you download regularly updated mbtiles files if you're interested: https://www.maptiler.com and https://stadiamaps.com.
